components/rootless_studio/publish-to-dockerhub.sh

```diff
@@ -13,7 +13,6 @@ if [[ -z $DOCKER_LOGIN_PASSWORD ]]; then
 fi
 
 docker login -u="$DOCKER_LOGIN_USER" -p="$DOCKER_LOGIN_PASSWORD"
-trap 'rm -f $HOME/.docker/config.json; exit $?' INT TERM EXIT
```

Comment:
No, I mean I don't think you need to manually manage the exit status; I think the `trap` itself is still probably useful.

Reply:
If you trap, won't the program exit with whatever command trap ran?

Reply:
```sh
bash-4.4$ cat ./traptest.sh
#!/bin/bash

exit_with=${1}

trap 'rm foo' EXIT
exit $exit_with
bash-4.4$ rm foo
rm: foo: No such file or directory
bash-4.4$ echo $?
1
bash-4.4$ ./traptest.sh 3
rm: foo: No such file or directory
bash-4.4$ echo $?
3
bash-4.4$ ./traptest.sh 0
rm: foo: No such file or directory
bash-4.4$ echo $?
0
bash-4.4$ bash --version
GNU bash, version 4.4.23(1)-release (x86_64-apple-darwin16.7.0)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
bash-4.4$
```
hat-tip to @baumanj